### ISSUE TITLE
Notifications: "Like" button shouldn't open the app

### ISSUE THREAD

**Issue by hoverduck** [wordpress-mobile/WordPress-Android#4602]
### Expected behavior
When I click the Like button in a notification I expect to remain in whatever app I'm currently using

### Actual behavior
I'm taken out of my current activity and placed into the WordPress app's notification view.  I'd much rather just get a toast message indicating success/failure rather than having the whole app open up and interrupt whatever else I'm doing.  This can be especially jarring when I interact with the notification from my Android Wear watch, since then the next time I unlock my phone it's in a different app from where I left it.

I'm labelling the issue with _[Type] Bug_, but if others feel strongly that this is the desired behavior it can be re-labelled an _Enhancement_ and worked on later

### Steps to reproduce the behavior
Receive a notification of a comment, and click Like

**Comment by mzorz** [comment]
I wouldn't call it a bug, but rather an `incremental approach to make users life easier` 😁 Going the "magic" way means some big refactoring but I've already considered this so I went the fastest route first (just focusing on _which_ actions should be there and analyzed all that), then leaving refactoring and ability to work all this out on the background for a second phase.
While this could be prioritized (I strongly agree what you describe here as `Expected behavior` is the way to go, anyway), just conceptually I would say this is an enhancement.
tl;dr maybe it's fine either way, I'm going to work on it anyway :)

**Comment by mzorz** [comment]
Just a heads up - a solution to this is awaiting review in #4667 👍
